Sourcery/Generating/Generator.swift

```diff
@@ -25,7 +25,7 @@ private class TypesReflectionBox: NSObject {
     lazy var classes: [Type] = {
         return self.types.filter { type in
             let isNotClass = type is Struct || type is Enum || type is Protocol
-            return !isNotClass
+            return !isNotClass && !type.isExtension
```

Comment:
Yes, `types.based` will also work, they only be filtered form `types.inheriting` which makes sense as we can only add protocol conformance with extensions.